homework.md

```diff
@@ -44,31 +44,31 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language for the website
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+When people want to share code to each other they can put it on Github,collaboration and 
```

Comment:
Weldone Vicky, We do not only share codes on Github, it could be any kind of document. You can as well mention the fact that it is used for version control i.e keeps track of every singlr change made.